src/query/test/compatibility/test.go

```diff
@@ -560,7 +562,7 @@ func almostEqual(a, b float64) bool {
 	diff := math.Abs(a - b)
 
 	if a == 0 || b == 0 || diff < minNormal {
-		return diff < epsilon*minNormal
+		return diff < epsilon
```

Comment:
nit: can delete `minNormal` completely I think

Reply:
Not sure about this. It seems that this is designed to catch near-zero values in these tests:
https://github.com/m3db/m3/blob/master/src/query/test/testdata/aggregators.test#L97

Reply:
Ah yeah missed that it was used on :564